acceptor_unix.go

```diff
@@ -36,6 +36,7 @@ func (svr *server) acceptNewConnection(fd int) error {
 		if err == unix.EAGAIN {
 			return nil
 		}
+		svr.opts.Logger.Warnf("failed to accept with error: %v", err)
```

Comment:
"failed to accept with error: %v" --> "Accept() fails due to error: %v"

Reply:
done.